### PR TITLE
Qa first session

### PR DESCRIPTION
**Pull request by choosen**
Sortable users table. 
Training state legend @ Popover. 
Add link titles on eventRender @ calendar. 
Rename to squash DEAL
fixed footer position.
Reconfig FullCalendar, RSpec. 
Change redirect after training and to app invitation. 
panelize lists, improve layout, 
apply some changes advised by QA

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 class UsersController < ApplicationController
+  helper_method :sort_column, :sort_direction
+
   before_action :authenticate_user!
   load_and_authorize_resource
 
+  def index
+    @users = User.order(sort_column + ' ' + sort_direction)
```
Comment (weirdobeaver) [inline app/controllers/users_controller.rb]: ladniej wyglada `[sort_column, sort_direction].join(' ')`

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -1,34 +1,12 @@
-# frozen_string_literal: true
```
Comment (weirdobeaver) [inline spec/rails_helper.rb]: a to chyba akurat moglo zostac :)

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -1,34 +1,12 @@
...
-# Prevent database truncation if the environment is production
 abort('The Rails env is running in production mode!') if Rails.env.production?
 require 'spec_helper'
 require 'rspec/rails'
 require 'devise'
 require 'factory_girl_rails'
-require 'support/controller_macros'
-require 'support/factory_without_validation'
-# Add additional requires below this line. Rails is not loaded until this point!
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```
Comment (weirdobeaver) [inline spec/rails_helper.rb]: :+1:

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (choosen) [inline app/controllers/users_controller.rb]: :+1:

File: spec/rails_helper.rb (hunk 2 above)
Comment (choosen) [inline spec/rails_helper.rb]: true , optymalniejsze mo

**Pull request merged**